microservices/gateway/src/middlewares/webhook.ts

```diff
@@ -0,0 +1,42 @@
+import { Log } from '@lomray/microservice-helpers';
+import { Gateway } from '@lomray/microservice-nodejs-lib';
+import { Response } from 'express';
+import type { RequestHandler } from 'express-serve-static-core';
+import remoteConfig from '@config/remote';
+
+/**
+ * Handles webhooks from other services
+ */
+const webhook =
+  (): RequestHandler =>
+  async (req, res, next): Promise<Response | void> => {
+    // @ts-ignore
+    const { headers, url, rawBody } = req;
+    const { webhookOptions } = await remoteConfig();
```

Comment:
```suggestion
    const { webhookOptions: { url: webhookUrl, allowMethods } } = await remoteConfig();
```

webhookOptions - make destructurisation